packages/reports/src/Stores/Modules/Statement/Helpers/format-response.ts

```diff
@@ -1,36 +1,42 @@
 import { formatMoney, toTitleCase, toMoment, getMarketInformation, getSymbolDisplayName } from '@deriv/shared';
 import { localize } from '@deriv/translations';
+import { ActiveSymbols, Statement } from '@deriv/api-types';
 
-export const formatStatementTransaction = (transaction, currency, active_symbols = []) => {
+export const formatStatementTransaction = (
+    transaction: NonNullable<Statement['transactions']>[number],
+    currency: string,
+    active_symbols: ActiveSymbols = []
+) => {
+    const { action_type, app_id, contract_id, longcode, purchase_time, withdrawal_details } = transaction;
     const format_string = 'DD MMM YYYY HH:mm:ss';
     const transaction_time = toMoment(transaction.transaction_time).format(format_string);
-    const payout = parseFloat(transaction.payout);
-    const amount = parseFloat(transaction.amount);
-    const balance = parseFloat(transaction.balance_after);
+    const payout = transaction.payout ?? NaN;
+    const amount = transaction.amount ?? NaN;
+    const balance = transaction.balance_after ?? NaN;
     const should_exclude_currency = true;
-    const shortcode = ['buy', 'sell'].includes(transaction.action_type) ? transaction.shortcode : null;
+    const shortcode = ['buy', 'sell'].includes(action_type ?? '') ? transaction.shortcode : null;
     const display_name = shortcode
         ? getSymbolDisplayName(active_symbols, getMarketInformation(shortcode).underlying)
         : '';
 
     return {
-        action: localize(toTitleCase(transaction.action_type) /* localize-ignore */), // handled in static_strings_app.js: 'Buy', 'Sell', 'Deposit', 'Withdrawal'
```

Comment:
`static_strings_app.js` doesn't exist anymore and this `action` is localized despite the `ignore` comment. hence, removing it.